src/webgl/framebuffer.js

```diff
@@ -241,14 +265,20 @@ export default class Framebuffer extends Resource {
   readPixels({
     x = 0,
     y = 0,
-    width,
-    height,
+    width = this.width,
+    height = this.height,
     format = GL.RGBA,
-    type,
-    pixelArray = null
+    type, // Auto deduced from pixelArray or gl.UNSIGNED_BYTE
+    pixelArray = null,
+    attachment = GL_COLOR_ATTACHMENT0 // TODO
   }) {
     const {gl} = this;
 
+    // TODO - Move into readPixels
+    if (attachment === GL.COLOR_ATTACHMENT0 && this.handle === null) {
+      attachment = GL.FRONT;
```

Comment:
> attachment = GL.FRONT;

`attachment` seem to be not used any where, in default case, following `this.bind()` is going to bund `null` handle any way, why do we need to handle this separately ?